src/schemas/21-4142/schema.js

```diff
@@ -0,0 +1,78 @@
+import definitions from '../../common/definitions';
+import _ from "lodash";
+
+
+let schema = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'AUTHORIZATION TO DISCLOSE INFORMATION TO THE DEPARTMENT OF VETERANS AFFAIRS (21-4142/21-4142a)',
+  type: 'object',
+  additionalProperties: false,
+  definitions: _.pick(definitions, [
+    'fullName',
+    'ssn',
+    'vaFileNumber',
+    'date',
+    'address',
+    'phone',
+    'dateRange',
+    'privacyAgreementAccepted'
+  ]),
```

Comment:
any reason why you stopped using `schemaHelpers.addDefinitionToSchema`? it will add the definition here and add the field in `properties`

Reply:
I can change this back to using addDefinitionToSchema for the handful of fields. This was the 8th time I had written the schema in an attempt to follow best practices and ensure the tests passed. In doing so this was just where I ended up with the implementation.